.github/workflows/build_publish.yml

```diff
@@ -0,0 +1,74 @@
+name: Build & Publish atomfeed-console
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**.md"
+
+jobs:
+  Trivy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Trivy Scan
+        run: ./.github/trivy_scan.sh .
+
+  build-atomfeed-console:
+    name: Build atomfeed-console
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Java 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: "zulu"
+          java-version: "8"
+      # - name: Run Unit Tests
+      #   run: mvn --no-transfer-progress clean test
+      - name: Install bower
```

Comment:
do we need bower?

Reply:
Yes Angshu, we need it. Without bower the build fails